nano/node/transport/transport.cpp

```diff
@@ -218,61 +218,12 @@ bool nano::transport::reserved_address (nano::endpoint const & endpoint_a, bool
 
 using namespace std::chrono_literals;
 
-nano::bandwidth_limiter::bandwidth_limiter (const size_t limit_a) :
-next_trend (std::chrono::steady_clock::now () + 50ms),
-limit (limit_a)
+nano::bandwidth_limiter::bandwidth_limiter (const size_t limit_burst_ratio_a, const size_t limit_a) :
```

Comment:
`limit_burst_ratio_a` could be floating point, with changes required in the config.